app/ios/ExportOptions.plist

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>method</key>
+	<string>app-store</string>
+	<key>provisioningProfiles</key>
+	<dict>
+		<key>com.pikatorrent.PikaTorrent</key>
+		<string>github_actions_ios</string>
```

Comment:
Just for my curiosity, why do we need that?

Reply:
This is the identifier/name of the profile to use. It is installed in the CI at the `Install the provisioning profile` step.